wpeverest/user-registration
# UR-1204 jQuery validation library updated to 1.19.5

### All Submissions:






### Changes proposed in this Pull Request:


This PR will add the latest version jQuery validation library.

### How to test the changes in this Pull Request:

1. Switch to this branch and check whether form validation is working properly or not.

### Types of changes:




### Other information:




### Changelog entry

> UR-1204 jQuery validation library updated to 1.19.5.
